packages/core/types/index.d.ts

```diff
@@ -47,11 +47,13 @@ export interface GlobalProps<Theme> {
  */
 export function Global<Theme = any>(props: GlobalProps<Theme>): ReactElement
 
-export function keyframes(
+export function keyframes<Theme = any>(
```

Comment:
Here down also had the same issue.

Reply:
Is it possible to access Theme inside `keyframes`? not sure how this can be done in vanilla JS